src/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,101 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+import React from 'react'
+import styled from 'styled-components'
+
+import { componentName } from '../../utils'
+
+import {
+  AddSWPRToMetamaskButton,
+  Base,
+  ButtonConfirmed as ButtonConfirm,
+  ButtonConfirmedProps,
+  ButtonDark1,
+  ButtonDark2,
+  ButtonEmpty,
+  ButtonError as ButtonErr,
+  ButtonErrorProps,
+  ButtonGrey,
+  ButtonInvisbile,
```

Comment:
There is a typo on the button name.
```suggestion
  ButtonInvisible,
```